server/protocols/jmap/src/test/java/org/apache/james/jmap/model/MessagePreviewGeneratorTest.java

```diff
@@ -64,4 +64,12 @@ public void computeShouldReturnStringIsLimitedTo256Length() throws Exception {
             .hasSize(MessagePreviewGenerator.MAX_PREVIEW_LENGTH)
             .isEqualTo(expected);
     }
+
+    @Test
+    public void computeShouldReturnNormalizeSpaceString() throws Exception {
```

Comment:
I used the StringUtils function which defines 
`S (white space) consists of one or more space (#x20) characters, carriage returns, line feeds, or tabs.
White Space
[3]   	S	   ::=   	(#x20 | #x9 | #xD | #xA)+`